webapp/tests/karma/ts/modules/tasks/tasks.component.spec.ts

```diff
@@ -43,6 +57,15 @@ describe('TasksComponent', () => {
     contactTypesService = {
       includes: sinon.stub(),
     };
+    sessionService = {
+      isDbAdmin: sinon.stub().returns(false),
+      isOnlineOnly: sinon.stub().returns(false),
+      userCtx: sinon.stub().returns({ name: 'Sarah' })
```

Comment:
Do we actually need to stub `isDbAdmin` and `userCtx` here?  I do not remember seeing these functions being used....